test/functionalTest/cases/0876_entity_dates/entity_dates_overriden_by_user.test

```diff
@@ -0,0 +1,415 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
overridden (in file name; probably inside test file as well)

Reply:
Fixed in commits 4274c664e11888879464ab84fde8d3f34768bdaf...ad4c3fcfee9b12d1b3ba0bfa62491b562f3cdabd